### PR TITLE
Add 'user' field at Impala datasource, and use it at connection.cursor

### PR DESCRIPTION
**Pull request by TaylorHere**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

## Description
At our use case, we did't use any auth mechanism, just a user pram at cursor function

### CODE REVIEW

File: redash/query_runner/impala_ds.py [hunk 1]
```diff
@@ -95,10 +96,12 @@ def run_query(self, query, user):
...
-            connection = connect(**self.configuration.to_dict())
-
-            cursor = connection.cursor()
-
+            config = self.configuration.to_dict()
+            connection = connect(**config)
+            if 'user' in config:
+                cursor = connection.cursor(user=config['user'])
+            else:
+                cursor = connection.cursor()
```
Comment (susodapop) [inline redash/query_runner/impala_ds.py:104]: What happens if there are both `ldap_user` and `user` fields? Does this fail gracefully?

File: redash/query_runner/impala_ds.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def configuration_schema(cls):
                 "ldap_user": {"type": "string"},
                 "ldap_password": {"type": "string"},
                 "timeout": {"type": "number"},
+                "user": {"type": "string"},
```
Comment (susodapop) [inline redash/query_runner/impala_ds.py:59]: I wonder how we can communicate to users when to use `user` versus `ldap_user`?

**Review by susodapop — COMMENTED** [review]
Thanks for your contribution! A couple questions below.

File: redash/query_runner/impala_ds.py (hunk 2 above)
Comment (TaylorHere) [reply redash/query_runner/impala_ds.py:59]: I thought that 'ldap_user' and 'user' should work separately, maybe I should add this condition test

File: redash/query_runner/impala_ds.py (hunk 1 above)
Comment (TaylorHere) [reply redash/query_runner/impala_ds.py:104]: @susodapop  hi there, I update the code~

**Pull request closed without merging**